### DISCUSSION TITLE
ReDoc - how to remove field name in brackets from parameter type?

### DISCUSSION THREAD

**Discussion by LeeLeman** [fastapi/fastapi#6067] [Questions] [answered]
### First Check

- [X] I added a very descriptive title to this issue.
- [X] I used the GitHub search to find a similar issue and didn't find it.
- [X] I searched the FastAPI documentation, with the integrated search.
- [X] I already searched in Google "How to X in FastAPI" and didn't find any information.
- [X] I already read and followed all the tutorial in the docs and didn't find an answer.
- [X] I already checked if it is not related to FastAPI but to [Pydantic](https://github.com/samuelcolvin/pydantic).
- [X] I already checked if it is not related to FastAPI but to [Swagger UI](https://github.com/swagger-api/swagger-ui).
- [X] I already checked if it is not related to FastAPI but to [ReDoc](https://github.com/Redocly/redoc).

### Commit to Help

- [X] I commit to help with one of those options 👆

### Example Code

```python
class CustomerTracking(Entity):
    customer_ip_address: str = Field(example="XXX.XXX.XXX.XXX")
    customer_user_agent: str = Field(example="Chrome")
```

### Description

I need to remove field names in brackets from parameter types.
For example: my model has string field `customer_ip_address` and i want to see type `string` in the ReDoc documentation, not `string (Customer Ip Address)`
Is there any setting to do this?

### Operating System

Linux

### FastAPI Version

0.88.0

### Python Version

3.10.6

### Additional Context

![image](https://user-images.githubusercontent.com/31480109/212621790-03cb192c-9dd3-44d8-9772-cbb8c0d0288c.png)

**Comment by mhmdibrahimm** [comment, marked as answer]
~~Only thing I found was to do this:~~
```py
class CustomerTracking(Entity):
    customer_ip_address: str = Field(example="XXX.XXX.XXX.XXX", title="")
    customer_user_agent: str = Field(example="Chrome", title="")
```

~~I still didn't find a way to disable them by default, so now I have to do each field manually.~~

EDIT: I got another workaround:
```py
from pydantic import Field as PydanticField

# I am defining the Field function with title already set as empty text.
def Field(**kwargs):
    return PydanticField(
        title="",
        default=None, # keep this if you want to remove the "required" red text under the field name.
        **kwargs,
    )


```
